07-asymmauth.tex

```diff
@@ -191,8 +191,8 @@ \subsection{Schlüssel- und Signaturerzeugung}
 Zahl $e \in \{1, \dots, p - 1\}$ gewählt,
 wobei $p=|\G|$. Damit berechnet man 
 \begin{align*}
-a &:= g^e \in \mathbb{G}\\
-b &:= (M - a \cdot x) \cdot e^{-1} \mod p
+a &:= g^e \mod p \in \mathbb{G}\\
```

Comment:
p=|G| ist hier die Ordnung der Gruppe. Allerdings müsste man hier modulo die Primzahl rechnen, oder nicht?
Bzw p ist hier doppeldeutig, da einmal in Z_p verwendet und dann als Gruppenordnung redefiniert.
Und unten dann modulo gruppenordnung also mod p